src/test/java/net/datafaker/FakerTest.java

```diff
@@ -283,6 +286,8 @@ void fakerInstanceCanBeAcquiredViaUtilityMethods() {
         assertThat(new Faker(Locale.CANADA)).isInstanceOf(BaseFaker.class);
         assertThat(new Faker(new Random(1))).isInstanceOf(BaseFaker.class);
         assertThat(new Faker(Locale.CHINA, new Random(2))).isInstanceOf(BaseFaker.class);
+        assertThat(new Faker(Locale.US, new RandomService())).isInstanceOf(BaseFaker.class);
+        assertThat(new Faker(new FakeValuesService(), new FakerContext(Locale.JAPAN, new RandomService()))).isInstanceOf(BaseFaker.class);
```

Comment:
what is the reason of such tests?
any `new Faker` as an instance of `BaseFaker`

Reply:
test coverage is not a good metric.
There is a much better metric to answer question if a test is a good one or not.
A good test is test which is failing otherwise it does not bring much value ...

Reply:
Now I get it, test is deleted